crates/net/network/src/transactions/fetcher.rs

```diff
@@ -190,7 +194,7 @@ impl TransactionFetcher {
             hashes_to_request.push(hash);
 
             // tx is really big, pack request with single tx
-            if size >= DEFAULT_SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE_ON_ASSEMBLE_GET_POOLED_TRANSACTIONS_REQUEST {
+            if size >= self.soft_limit_byte_size_pooled_transactions_response {
```

Comment:
here the correct constant has been replaced for example

Reply:
this is actually incorrect, should be `soft_limit_byte_size_pooled_transactions_response_on_pack_request`

Reply:
that comment was left before that variable existed in the code. spot on it should be `soft_limit_byte_size_pooled_transactions_response_on_pack_request`.